src/chigame/games/views.py

```diff
@@ -19,6 +22,28 @@ class LobbyListView(SingleTableView):
     template_name = "games/lobby_list.html"
 
 
+@login_required
+def lobby_join(request, pk):
+    lobby = get_object_or_404(Lobby, pk=pk)
+    joined = Lobby.objects.filter(members=request.user.id)
+    print(joined, lobby)
+    if lobby in joined:
+        return HttpResponseNotFound("Already joined.")
```

Comment:
I don't think it makes sense to redirect to a different page with "already joined". You should do something like delete the join button if the user has already joined. You can also do an error pop up on the same page with something like this 
```suggestion
if lobby in joined:
        messages.error(request, "Already joined.")
        return redirect(reverse("lobby-details", kwargs={"pk": lobby.id}))
```
This uses `from django.contrib import messages`

Reply:
thank you for the feedback. Fixed!